cg/services/post_processing/pacbio/data_transfer_service/utils.py

```diff
@@ -18,7 +18,8 @@ def get_sequencing_run_dto(metrics: PacBioMetrics) -> PacBioSequencingRunDTO:
         type=DeviceType.PACBIO,
         well=metrics.dataset_metrics.well,
         plate=metrics.dataset_metrics.plate,
-        movie_time_hours=None,
+        run_started_at=metrics.dataset_metrics.run_started_at,
+        run_completed_at=metrics.dataset_metrics.run_completed_at,
```

Comment:
Same as with the database model. Do we need run_xxxxx?

Reply:
implemented